protos/chaincode.proto

```diff
@@ -78,10 +78,16 @@ message ChaincodeSpec {
 // TODO: Define `codePackage`.
 message ChaincodeDeploymentSpec {
 
+    enum ExecutionEnvironment {
+        DOCKER = 0;
```

Comment:
I think it would be best if SYSTEM=0 and DOCKER=1, as additional execution environments beyond Docker may be available in the future.